docs/README.md

```diff
@@ -51,60 +59,46 @@ To deploy this pipeline to your own Azure environment, follow these steps.
 
-#### Infrastructure as Code
+### Serverless Functions
 
-Every resource required to use the Building Blocks and pipelines implemented in this respostory are defined using Terraform. This makes it simple for users to deploy all of the functionality provided in this repository to their own Azure environments. The Terraform code can be found in the `terraform` directory.
+The pipeline deployed by the Starter Kit processes data as it is received in real time. We achieve this event-driven behavior by using an Azure Function to “listen” for new data to be uploaded. Azure Functions are Azure's version of serverless functions, similar to Lambda in Amazon Web Services (AWS). Serverless functions provide a relatively simple way to run services with modest runtime duration, memory, and compute requirements in the cloud. The [`serverless-functions` directory](https://github.com/CDCgov/phdi-azure/tree/main/serverless-functions) contains the Python source code for this Azure Function.
 
-#### Continuous Integration and Continuous Deployment
+### Pipeline Orchestration
 
-In order to ensure high code quality and reliability we have implemented a Continuous Integation (CI) pipeline consisting of a suite of tests all new contributions must pass before they are merged into `main`. We have also built a Continuous Deployment (CD) pipeline that automatically deploys the code in the repositiory to linked Azure environments when changes are made. The combined CI/CD pipeline is implemented with GitHub Actions in the `.github` directory. 
+Since the Building Blocks are designed to be composable, users will likely want to chain several together into pipelines. We use Microsoft's Azure Data Factory (ADF) to define the process that requires the use of multiple Building Blocks. These pipelines can be configured using the ADF interface (web UI).
 
-### Target Users
+### Infrastructure as Code
 
-Target users of this system include:
+Every resource required to use the Building Blocks and pipelines implemented in this repository are defined using Terraform. This makes it simple for users to deploy all of the functionality provided in this repository to their own Azure environments. The Terraform code can be found in the [`terraform` directory](https://github.com/CDCgov/phdi-azure/tree/main/terraform).
 
-- Public Health Departments
-  - Epidemiologists who rely on health data to take regular actions
-  - Senior stakeholders who make executive decisions using aggregate health data
-  - IT teams who have to support epidemiologists and external stakeholders integrating with the PHD
-  - PHDs may include state, county, city, and tribal organizations
-- CDC
-  - Employees and contractors working on CDC projects with access to a Azure environment and interest in using PHDI Building Blocks
+### Continuous Integration and Continuous Deployment
```

Comment:
I actually just moved this section to a separate doc ([devops.md](https://github.com/CDCgov/phdi-azure/blob/main/docs/devops.md)) so this CI/CD section can be removed!

Reply:
I think we should still have a section on this in the README, but instead it will link to the devops.md doc that you created instead of the github directory